backend/inspirehep/records/api/jobs.py

```diff
@@ -16,3 +18,29 @@ class JobsRecord(InspireRecord):
     es_serializer = JobsElasticSearchSchema
     pid_type = "job"
     pidstore_handler = PidStoreJobs
+
+    @staticmethod
+    def get_jobs_by_deadline(date):
+        """Return a list of all open jobs which deadline match the given date.
+
+        Args:
+            date (str): string representing a date in the format YYYY-MM-DD.
+
+        Return:
+            list: list of open jobs matching a deadline.
+
+        """
+        query = {
+            'query':
+                {
+                    'bool':
+                        {
+                            'must': [
+                                {'match': {'deadline_date': date}},
+                                {'match': {'status': 'open'}}
+                            ]
+                        }
+                }
+        }
+        result = es.search(index='records-jobs', body=query)
+        return [r['_source'] for r in result['hits']['hits']]
```

Comment:
do you really need to do this manually? there's no helper to get a list of records from a search?

Reply:
Does not look like.

Reply:
https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/search/api.py#L181 ?

Reply:
I see, I thought you were talking about some helper from the Python ES library.